mautic/mautic
# Allow files in a form submission to be send as attachments

This PR is rebase of https://github.com/mautic/mautic/pull/7707 for 3.x

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | Y 
| Automated tests included? | N
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | N
| Deprecations? | N

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
We use forms heavily for all of our customers. Sometimes users can upload attachments, for example a CV when applying for a job. These form submissions are then sent to HR, but these people don't necessarily have access to Mautic so they can't download or read the CV.

This PR provides a simple solution: You can configure Mautic to instead attach all files to the email, so it can be easily sent/forwarded to anybody.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. 
2. 

#### Steps to test this PR:

1. Load up [this PR](https://m3.mautibox.com/)
2. Create a form with a file field
3. Add an action "Send form result" and enable the checkbox "Attach file in email"
4. Go to preview and submit the form
5. Check your emails

### Interface in Mautic
<img width="913" alt="New_Form___Mautic" src="https://user-images.githubusercontent.com/2618452/79530812-0e309e00-808e-11ea-9fb7-dafc8ec614e3.png">

#### List deprecations along with the new alternative:
1. 
2. 

#### List backwards compatibility breaks:
1. 
2.
